Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ruby:2.7
+
+WORKDIR /usr/src/app
+
+COPY Gemfile Gemfile.lock ./
+
+RUN bash -c "echo 'deb http://us.archive.ubuntu.com/ubuntu focal main restricted universe' | tee /etc/apt/sources.list; apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 3B4FE6ACC0B21F32; apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 871920D1991BC93C; apt update"
+
+RUN apt install -y libpq-dev
+
+RUN bundle install
```

Comment:
You should be able to filter down to `bundle install --deployment --without development test` in order to bundle for production and leave out all of the development dependencies such as spellcheck, etc).

Reply:
Also, somewhere early on in this file there should be the line which should handle the database environment automatically.

`ENV RAILS_ENV=production`